src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -451,14 +469,25 @@ def update(
                 "nqn": Param(str, "NVMeoF subsystem NQN"),
                 "nsid": Param(str, "NVMeoF Namespace ID"),
                 "gw_group": Param(str, "NVMeoF gateway group", True, None),
+                "force": Param(str, "Force remove the RBD image")
             },
         )
         @NvmeofCLICommand("nvmeof ns del")
         @empty_response
         @handle_nvmeof_error
-        def delete(self, nqn: str, nsid: str, gw_group: Optional[str] = None):
+        def delete(
+            self,
+            nqn: str,
+            nsid: str,
```

Comment:
why are we taking nsid as string ? Why not int ?
I am seeing this in every API

Reply:
That's a good question. @epuertat do you remember why it was like this? or was that a behavior that you carried from my initial implementation?